packages/mobile/src/screens/notifications-screen/Notifications/RemixContestStartedNotification.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import { useTrack, useUser } from '@audius/common/api'
+import type { RemixContestStartedNotification as RemixContestStartedNotificationType } from '@audius/common/store'
+
+import { IconTrophy } from '@audius/harmony-native'
+import { useNotificationNavigation } from 'app/hooks/useNotificationNavigation'
+
+import {
+  NotificationHeader,
+  NotificationText,
+  NotificationTile,
+  NotificationTitle,
+  EntityLink,
+  UserNameLink
+} from '../Notification'
+
+const messages = {
+  title: 'New Remix Contest',
+  description: 'started a remix contest for'
+}
+
+type RemixContestStartedNotificationProps = {
+  notification: RemixContestStartedNotificationType
+}
+
+export const RemixContestStartedNotification = (
+  props: RemixContestStartedNotificationProps
+) => {
+  const { notification } = props
+  const { entityId, entityUserId } = notification
+
+  const navigation = useNotificationNavigation()
+  const { data: user } = useUser(entityUserId)
+  const { data: track } = useTrack(entityId)
+
+  const handlePress = useCallback(() => {
+    if (track) {
+      navigation.navigate(notification)
+    }
+  }, [track, navigation, notification])
+
+  if (!user || !track) return null
+
+  return (
+    <NotificationTile notification={notification} onPress={handlePress}>
+      <NotificationHeader icon={IconTrophy}>
+        <NotificationTitle>{messages.title}</NotificationTitle>
+      </NotificationHeader>
+      <NotificationText>
+        <UserNameLink user={user} /> {messages.description}{' '}
```

Comment:
interesting spacing

Reply:
yeah this is how the other notifs look too i don't like it but 🤷

Reply:
just jsx and linting clashing weird, it's chill